### ISSUE TITLE
Can you provide detailed documentation for deployment to kubernetes?

### ISSUE THREAD

**Issue by liguodongIOT** [ploomber/projects#5]
I failed to install an existing document deployment.
Can you provide deployment model training and model inference that are deployed in kubernetes?

**Comment by edublancas** [comment]
Hi,

check out the kubernetes example in the documentation https://soopervisor.readthedocs.io/en/latest/tutorials/kubernetes.html

**Comment by liguodongIOT** [comment]
> Hi,
> 
> check out the kubernetes example in the documentation https://soopervisor.readthedocs.io/en/latest/tutorials/kubernetes.html

Hi, I found this document a bit out of date.

for example:

`https://raw.githubusercontent.com/argoproj/argo/stable/manifests/quick-start-postgres.yaml` ,

This url has been changed。The new url is `https://raw.githubusercontent.com/argoproj/argo-workflows/stable/manifests/quick-start-postgres.yaml`。

and when I run this command `soopervisor add training --backend argo-workflows`  ,an error occurs.

> Error: Expected environment.lock.yml or requirements.txt.lock at the root directory. Add one.

How should I generate environment.lock.yml or requirements.txt.lock file?
